pyms/flask/services/service_discovery.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import uuid
+
+try:
+    import consulate
+except ModuleNotFoundError:  # pragma: no cover
+    consulate = None
+
+from pyms.constants import LOGGER_NAME
+from pyms.flask.services.driver import DriverService
+from pyms.utils import import_from
+
+logger = logging.getLogger(LOGGER_NAME)
+
+CONSUL_SERVICE_DISCOVERY = "consul"
+
+EUREKA_SERVICE_DISCOVERY = "eureka"
```

Comment:
It should be removed as not currently supported.

Reply:
Done!